chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb

```diff
@@ -49,17 +49,26 @@ touch $RUNFILE
 trap cleanup INT EXIT
 
 <% if @use_ha %>
-# Shutdown pacemaker so the remaining OpenStack services are stopped
+# Shutdown corosync so the remaining OpenStack services are stopped
 # This is needed so that the zypper dup won't trigger any db migrations on package update
-service pacemaker stop
+service corosync stop
 
 ret=$?
 if [ $ret != 0 ] ; then
-    log "Error occured during shutting down pacemaker service"
+    log "Error occured during shutting down corosync service"
     echo $ret > $UPGRADEDIR/crowbar-pre-upgrade-failed
     exit $ret
 fi
 
+# Make sure sbd is shut down (if it was configured) before proceeding with next actions
+if systemctl is-enabled sbd 2>/dev/null ; then
```

Comment:
Just redirecting stderr will still produce output stuff on stdout when the service is enabled. So we might still need ```--quiet``` here.